libs/crosstarget-utils/src/common.rs

```diff
@@ -21,3 +21,24 @@ impl Display for TimeoutError {
 }
 
 impl std::error::Error for TimeoutError {}
+
+#[derive(Debug)]
+pub struct RegExpError {
+    pub message: String,
+}
+
+#[derive(PartialEq)]
+pub enum RegExpFlags {
```

Comment:
nit: maybe `RegExpFlag` in singular instead? `RegExpFlags` sounds more like a name for a type that represents a bitmask of multiple flags

Reply:
Yeah, let's have a bitmask, looking at the code further